.nengobones.yml

```diff
@@ -32,6 +32,7 @@ setup_py:
     - numpydoc>=0.6
     - nengo_sphinx_theme>=0.12.0
   tests_req:
+    - nengo[tests]>=3.0.0
     - pytest>=3.6
```

Comment:
This is included in `nengo[tests]`

Reply:
Yup. But I left this in because this "fix" is meant to be temporary. The plan is to split off the Nengo pytests into another package so that just importing nengo in the nengo-fpga tests doesn't require all of the packages that the full `nengo[tests]` requires. So, I added the extra line here to patch the import error, and when the Nengo pytests have been split off, I'll remove the `nengo[tests]` line from here.